tests/python/unittest/test_runtime_rpc.py

```diff
@@ -45,7 +45,7 @@
 # run individual functions. Somewhere along the way, the imports are being
 # lost, so the server ends up not registering the functions.
 pytestmark = pytest.mark.skipif(
-    multiprocessing.get_start_method() != "fork",
+    sys.platform.startswith("win") == False and multiprocessing.get_start_method() != "fork",
```

Comment:
Could you add a comment here why Windows is being excluded here?

Reply:
Actually this enables this test for Windows. Windows does not support fork so this test could never run on Windows, but now it can.